Tools/autotest/arduplane.py

```diff
@@ -3291,7 +3291,23 @@ def fly_landing_baro_drift(self):
         self.wait_ready_to_arm()
         self.arm_vehicle()
 
-        self.fly_mission("ap-circuit.txt", mission_timeout=1200)
+        # Load and start mission
+        self.load_mission("ap-circuit.txt", strict=True)-1
+        self.set_current_waypoint(0, check_afterwards=False)
+        self.change_mode('AUTO')
+        self.wait_current_waypoint(1, timeout=5)
+        self.wait_groundspeed(0, 30, timeout=5)
```

Comment:
Perhaps you should constrain this wait a bit more?  "anywhere between 0 and 30" is pretty wide?